torch/linalg/__init__.py

```diff
@@ -8,6 +8,76 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cholesky = _add_docstr(_linalg.linalg_cholesky, r"""
+linalg.cholesky(input, *, out=None) -> Tensor
+
+Computes the Cholesky decomposition of a Hermitian (or symmetric for real-valued matrices)
+positive-definite matrix or the Cholesky decompositions for a batch of such matrices.
+The returned matrix ``L`` is lower-triangular, and
```

Comment:
This needs to be a little clearer about the batching behavior. I think a very small tweak and a reordering should do it:

Each decomposition has the form:

math goes here

where L is a lower-triangular matrix and ...